dev/travis/script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright © 2015 Magento. All rights reserved.
+# See COPYING.txt for license details.
+
+set -e
+export PATH="./../../../vendor/bin:$PATH"
+
+case $TEST_SUITE in
+    unit)
+        cd dev/tests/unit
+        phpunit
+        ;;
+    integration)
+        cd dev/tests/integration
+        phpunit
+        ;;
+    static)
+        cd dev/tests/static
+        phpunit --filter 'Magento\\Test\\Php\\LiveCodeTest::(testCodeStyle|testAnnotationStandard)'
```

Comment:
Interesting, that @mazhalai has also splitted them... :) while we were discussing it here.
Let's leave it as is then and we'll merge all together, when will be merging the PR and internal changed together.
Sorry for this long discussion.